Earthfile

```diff
@@ -341,22 +341,25 @@ earthly:
     # is particularly useful for disabling optimizations to make the binary work
     # with delve. To disable optimizations:
     #
-    #     -GO_GCFLAGS='all=-N -l'
+    #     --GO_GCFLAGS='all=-N -l'
     ARG GO_GCFLAGS
     ARG EXECUTABLE_NAME="earthly"
     ARG DEFAULT_INSTALLATION_NAME="earthly-dev"
     RUN test -n "$GOOS" && test -n "$GOARCH"
     RUN test "$GOARCH" != "arm" || test -n "$VARIANT"
     ARG EARTHLY_TARGET_TAG_DOCKER
-    ARG VERSION="dev-$EARTHLY_TARGET_TAG_DOCKER"
+    # TAG is the container image tag whereas BIN_VERSION is the version
```

Comment:
is this change still needed? I don't easily follow why we need these values to be different.

Reply:
Good point. I don't think it's needed with the `SAVE IMAGE` approach.

Reply:
Updated. I also reverted `TAG` back to `VERSION` as it's more suitable for `+earthly`.